README.md

```diff
@@ -57,22 +62,16 @@ the final APPROVER is still ``a11r``.
 ## Proposal Categories
 The proposals shall be numbered in increasing order.
 
-- ``#An`` - Affects all languages.
-- ``#Pnn`` - Affects processes, such as the proposal process itself.
-- ``#Lnnn`` - Language specific changes to external APIs or platform support.
-- ``#Gnnnn`` - Protocol level changes.
+- ``An`` - Affects all languages.
+- ``Pnn`` - Affects processes, such as the proposal process itself.
+- ``Lnnn`` - Language specific changes to external APIs or platform support.
+- ``Gnnnn`` - Protocol level changes.
+
+## Updating proposals
+When updating a proposal, the PR description should be named as follows:
```

Comment:
Maybe we should say something about when and why we would update an existing gRFC instead of creating a new one?  For example, maybe something like "Sometimes, after a gRFC is approved, small changes are needed to the design.  In these cases, it is often simpler to modify the existing gRFC rather than create a new one."

Reply:
Done (with slightly different wording).